File: codingTest/programmers/level_1/K번째수.js
function solution(array, commands) {
  var answer = [];
  commands.forEach(el =>
    answer.push(array.slice(el[0] - 1, el[1]).sort((a, b) => a - b)[el[2] - 1]),
  );
  return answer;
}

console.log(
  solution(
    [1, 5, 2, 6, 3, 7, 4],
    [
      [2, 5, 3],
      [4, 4, 1],
      [1, 7, 3],
    ],
  ),
);

File: codingTest/programmers/python/k번째수.py
def solution(arr, commands):
    result = []
    for li in commands:
        result.append(sorted(arr[li[0] - 1 : li[1]])[li[2] - 1])
    return result


print(solution([1, 5, 2, 6, 3, 7, 4], [[2, 5, 3], [4, 4, 1], [1, 7, 3]]))
